scale_build/image/update.py

```diff
@@ -0,0 +1,147 @@
+import contextlib
+import glob
+import itertools
+import logging
+import os
+import textwrap
+import shutil
+
+from scale_build.config import SIGNING_KEY, SIGNING_PASSWORD
+from scale_build.utils.manifest import get_manifest
+from scale_build.utils.run import run
+from scale_build.utils.paths import CHROOT_BASEDIR, CONF_SOURCES, RELEASE_DIR, UPDATE_DIR
+
+from .manifest import build_manifest, build_update_manifest, UPDATE_FILE, UPDATE_FILE_HASH
+from .utils import run_in_chroot
+
+
+logger = logging.getLogger(__name__)
+
+
+def build_rootfs_image(update_image_logger):
+    for f in glob.glob(os.path.join('./tmp/release', '*.update*')):
+        os.unlink(f)
+
+    shutil.rmtree(UPDATE_DIR, ignore_errors=True)
```

Comment:
What errors are you ignoring?

We expect `UPDATE_DIR` to be at least empty after this method terminates successfully. Will it always be so?